### PR TITLE
refactoring test code

### PR DESCRIPTION
**Pull request by elektronaut0815**
No functional changes, just cleaning up.

### CODE REVIEW

File: test/tests.js [hunk 1]
```diff
@@ -142,57 +113,42 @@ contract('MagicBetTests', (accounts) => {
...
+    return userBalance;
+  }
+
   it('check market states transition', async () => {
     const marketStates = Object.freeze({WAITING: 0, OPEN: 1, LOCKED: 2, WITHDRAW: 3});
     // opening date in the future, so revert;  no need to increment state cos automatic within
     // the placeBet function via the checkState modifier
     expect((await magicBet.getCurrentState()).toNumber()).to.equal(marketStates.WAITING);
-    await expectRevert(placeBet(user0, NON_OCCURING, stake0), errors.incorrectState);
+    await expect(placeBet(user0, NON_OCCURING, stake0), errors.incorrectState).to.be.reverted;
```
Comment (gorgos) [inline test/tests.js:151]: Does this still test for the error message?

Comment (elektronaut0815) [reply test/tests.js]: fixed

File: test/tests.js [hunk 2]
```diff
@@ -201,13 +157,13 @@ contract('MagicBetTests', (accounts) => {
     await time.increase(time.duration.seconds(100));
     expect((await magicBet.getCurrentState()).toNumber()).to.equal(marketStates.LOCKED);
     // withdraw fail; too early
-    await expectRevert(magicBet.withdraw({from: user0}), errors.incorrectState); // too early
+    await expect(magicBet.withdraw({from: user0}), errors.incorrectState).to.be.reverted; // too early
```
Comment (gorgos) [inline test/tests.js:160]:
```suggestion
    await expect(magicBet.withdraw({from: user0})).to.be.revertedWith(errors.incorrectState); // too early
```

File: test/tests.js [hunk 3]
```diff
@@ -201,13 +157,13 @@ contract('MagicBetTests', (accounts) => {
...
     // determine winner then end
     await realitio.setResult(OCCURING);
     await magicBet.determineWinner();
     expect((await magicBet.getCurrentState()).toNumber()).to.equal(marketStates.WITHDRAW);
     await magicBet.withdraw({from: user0}); // should succeed now
-    await expectRevert(placeBet(user0, OCCURING, stake1), errors.incorrectState);
+    await expect(placeBet(user0, OCCURING, stake1), errors.incorrectState).to.be.reverted;
```
Comment (gorgos) [inline test/tests.js:166]:
```suggestion
    await expect(placeBet(user0, OCCURING, stake1)).to.be.revertedWith(errors.incorrectState);
```

File: test/tests.js [hunk 4]
```diff
@@ -278,7 +234,7 @@ contract('MagicBetTests', (accounts) => {
     await aToken.generate10PercentInterest(magicBet.address);
 
     // havnt waited a month so this should fail:
-    await expectRevert(magicBet.withdraw({from: user0}), errors.incorrectState);
+    await expect(magicBet.withdraw({from: user0}), errors.incorrectState).to.be.reverted;
```
Comment (gorgos) [inline test/tests.js:237]:
```suggestion
    await expect(magicBet.withdraw({from: user0})).to.be.revertedWith(errors.incorrectState);
```

File: test/tests.js [hunk 5]
```diff
@@ -291,7 +247,7 @@ contract('MagicBetTests', (accounts) => {
   it("can't determine winner if oracle has not yet resolved", async () => {
     await time.increase(time.duration.seconds(100));
     await placeBet(user0, NON_OCCURING, stake0);
-    await expectRevert(magicBet.determineWinner(), errors.oracleNotFinalised);
+    await expect(magicBet.determineWinner(), errors.oracleNotFinalised).to.be.reverted;
```
Comment (gorgos) [inline test/tests.js:250]:
```suggestion
    await expect(magicBet.determineWinner()).to.be.revertedWith(errors.oracleNotFinalised);
```

**Pull request merged**